src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -134,8 +139,11 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             // i.e. update project files and delete .sonarlint folder
             await MakeLegacyFileChangesAsync(legacySettings, changedFiles, progress, token);
 
+            // Trigger an refetch of suppressions so the Roslyn settings are updated.
+            await suppressionIssueStoreUpdater.UpdateAllServerSuppressionsAsync();
+
             progress?.Report(new MigrationProgress(0, 1, "Migration finished successfully!", false));
-            logger.WriteLine(MigrationStrings.Process_Finished);
+                logger.WriteLine(MigrationStrings.Process_Finished);
```

Comment:
if you are fixing the typo also fix the indent else ignore it

Reply:
Done